app/controllers/application_controller.rb

```diff
@@ -41,4 +42,21 @@ def orders_by_status(orders, status)
     orders.select { |order| order if order.status == status }
   end
 
+  def number_to_currency(price_in_cents)
+   return nil if price_in_cents.nil?
+   "$" + sprintf('%.2f', (price_in_cents / 100.0))
+  end
```

Comment:
Helper methods don't belong in the application controller. View helpers belong in the helpers directory. You can put controller and model helpers in the /lib directory.